src/nebula-hooks/use-sorting.ts

```diff
@@ -4,8 +4,12 @@ import { Column, HyperCube } from '../types';
 export const sortingFactory = (model: EngineAPI.IGenericObject | undefined, hyperCube: HyperCube) => {
   if (!model) return undefined;
 
-  return async (column: Column) => {
+  return async (column: Column, sortDirection?: string) => {
```

Comment:
@haxxmaxx I kept the sortDirection optional otherwise I should add this value wherever ChangeSortOrder has been used.

Reply:
Great, this is correct. It should behave differently when not passing an argument here